scripts/variantstore/AOU_DELIVERABLES.md

```diff
@@ -1,47 +1,35 @@
-    3. GvsCreateAltAllele
-    4. GvsCreateFilterSet
-        - make note of the "filter_set_name" input used
-    5. GvsPrepareCallset
-        - If the ```sample_info``` table contains control samples, create views in your dataset called ```sample_info_without_controls``` populated by the query ```SELECT * FROM `aou-genomics-curation-prod.beta_release.sample_info` where sample_id > 50``` and ```sample_info_controls_only``` populated by the query ```SELECT * FROM `aou-genomics-curation-prod.beta_release.sample_info` where sample_id <= 50```. A run for ```sample_info_without_controls``` will create the tables for the joint callset deliverable.  The run with ```sample_info_controls_only```  (with a distinct `destination_cohort_table_prefix` from the non-control run) will be used for the ["Sensitivity and Precision" deliverable](tieout/AoU_PRECISION_SENSITIVITY.md).
-        - If the callset has more than 20K genomes, you will want to set the input "skip_vet_new_insert" to `true` which will run all of the queries except the last one. It will print out the last query that needs to run with flex slots; allocate the flex slots, run the query, and then deallocate them.
-    6. GvsExtractCallset
-        - include a valid (and secure) "output_gcs_dir" parameter that the service account (see above) has access to, this is where the VCF and interval list files  will go
-        - use "fq_cohort_extract_table_prefix" without the project id or dataset name from `GvsPrepareCallset` for the input "extract_table_prefix"
-    7. Run [process to create callset statistics](callset_QC/README.md), for which you need
-        1. permission to query table `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites`
-        2. the data_project you used for all the GVS WDLs
-        3. the default_dataset you used for all the GVS WDLs
-        4. the "cohort_extract_table_prefix" input from `GvsExtractCallset` step
-        5. the "filter_set_name" input from `GvsCreateFilterSet` step
+## Prerequisites
+- If this is the first time running the GVS pipeline in a particular Google billing project, use your GCP account team to create a support ticket for the BigQuery team that includes "enabling cluster metadata pruning support for the BQ Read API." This enables a pre-GA feature that dramatically reduces the amount of data scanned reducing both cost and runtime.
+- This list assumes you are starting off with a Terra workspace that contains the re-blocked (version 2) gVCFs for your callset as "sample sets" in the "Data" tab.  You will also want to increase the Google quotas for the workspace project to these levels (all in region `us-central1`):
+    1. Persistent Disk Standard (GB): 1,000,000 GB (1 PB)
+    2. CPUs: 6,4000
+    3. In-use IP addresses: 5,000 (this is the most challenging one and will probably require contacting your GCP account team to facilitate)
```

Comment:
why is the number of IP addresses an order of magnitude lower than the CPU and VM numbers?

Reply:
Not sure — these are the quota increases we needed to run Beta and Charlie (based on when we'd bump up against them).  Probably has to do with how different resources are handed out.